autoload/airline/extensions.vim

```diff
@@ -336,6 +336,11 @@ function! airline#extensions#load()
     call add(s:loaded_ext, 'lsp')
   endif
 
+  if (get(g:, 'airline#extensions#nvimlsp#enabled', 1) && exists(':LspInstallInfo'))
```

Comment:
Since nvim-lsp only works with Neovim, I think we need to add this configuration.

```vim
if (get(g:, 'airline#extensions#nvimlsp#enabled', 1) && exists(':LspInstallInfo')) && has('nvim')
```

Reply:
yep, I've already added that. Please, check once again.